run_games.py

```diff
@@ -241,14 +238,34 @@ def main():
     features.add_argparse_args(parser)
     args = parser.parse_args()
 
+    stockfish_base = args.stockfish_base
+    stockfish_test = args.stockfish_test
+    if stockfish_test is None:
+        stockfish_test = stockfish_base
+
+    if not os.path.exists(stockfish_base):
+       sys.exit("Stockfish base does not exist!")
+
+    if not os.path.exists(stockfish_test):
+       sys.exit("Stockfish test does not exist!")
+
+    if not os.path.exists(args.ordo_exe):
```

Comment:
does this respect the `PATH` variable?

Reply:
perhaps do some `ordo --version` or an equivalent?

Reply:
OK, good idea, need to check.